front-end/sass/themes/color-stages.sass

```diff
@@ -16,7 +16,8 @@ $color-stages-gutter: 5px
             .status__image-container,
             .status__user-image-container,
             .status__detail-title,
-            .status__detail-sub-title
+            .status__detail-sub-title,
+            .status__detail-jobs-and-stages
```

Comment:
Instead of adding backgrounds to the specific elements, why not set it on the entire `.status__container` class?

Reply:
If i do that, i get a crappy container